lib/smart_todo_cop.rb

```diff
@@ -31,7 +31,7 @@ def smart_todo?(comment)
 
           metadata.events.any? &&
             metadata.events.all? { |event| event.is_a?(::SmartTodo::Parser::MethodNode) } &&
-            metadata.assignee
+            !metadata.assignees.empty?
```

Comment:
Nitpick, but I prefer to not use a negation `metadata.assignees.any?`

Reply:
Changed negation to `any?`